frontend/components/pages/CourseDetail.tsx

```diff
@@ -75,23 +95,28 @@ export function CourseDetail({
   })
 
   useEffect(() => {
-    setIsLoading(true)
+    setLoadingStates({ watching: true, favoriting: true })
```

Comment:
好みでもありますがboolean値は is~~~がわかりやすいです。

```suggestion
    setLoadingStates({ watching: true, isFavorite: true })
```

の方が良いかなぁ

Reply:
修正しました！